styleguide/_themes/global/scss/banners.scss

```diff
@@ -1,22 +1,31 @@
-.banner {
+@mixin make-banner {
   background-repeat: no-repeat;
   background-size: cover;
   color: $white;
   font-family: $heading;
+}
+
+$banner-height-large: 375px;
+$banner-height-medium: 225px;
+$banner-text-margin-large: 18%;
+$banner-text-margin-medium: 10%;
+
```

Comment:
this stuff is going to change for the imac banners but we will still have to support the legacy stuff. its going to get kind of interesting but this will eventually be suuuuuper useful i think.